scripts/docker/terminal-manager.ts

```
@@ -562,34 +568,102 @@ export class TerminalManager {
+        console.log(`${COLORS.GREEN}1.${COLORS.RESET} Copy cd command to clipboard`);
+        console.log(`${COLORS.GREEN}2.${COLORS.RESET} Create temporary script to source`);
+        console.log(`${COLORS.GREEN}3.${COLORS.RESET} Show terminal command to run manually\n`);
+
+        const choice = await this.prompt('Enter your choice (1-3): ');
+
+        switch (choice.trim()) {
+            case '1':
+                await this.copyToClipboard(`cd "${workingDir}"`, 'cd command');
+                break;
+            case '2':
+                const changeResult = await SystemUtils.changeDirectory(workingDir);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)